locales/zh_CN/LC_MESSAGES/start/download.po

```diff
@@ -137,20 +137,20 @@ msgstr "开发版"
 
 #: ../../doc/doc/download.rst:89
 msgid ""
-"`Nuitka 0.6.20rc3.zip "
-"<https://nuitka.net/releases/Nuitka-0.6.20rc3.zip>`__"
+"`Nuitka 0.6.20rc6.zip "
+"<https://nuitka.net/releases/Nuitka-0.6.20rc6.zip>`__"
 msgstr ""
 
 #: ../../doc/doc/download.rst:89
 msgid ""
-"`Nuitka 0.6.20rc3.tar.gz "
```

Comment:
These definitely should not appear for translation, what to do about them? Same for URLs.

Reply:
Ok, if possible it would be good to avoid it though, otherwise it's just a trap for others to fall into.

Reply:
This is the responsibility of `sphinx-intl`, so it is difficult to modify. Only by appointment.

Reply:
The approach I often take is filtering afterwards and wrap tools such that they behave sanely. I e.g. now monkey patch `sphinx-autobuild` such that it fits my use case. And the post-processing step is also something like that. If I tool doesn't let me do, I will do it, see search results.

For me, I will clean up stuff for a while, with translations being something straightforward to do and manage.